internal/commands/user/input_test.go

```diff
@@ -96,3 +99,191 @@ func TestProviderTypeDisplayUser(t *testing.T) {
+			},
+		} {
+			t.Run(tc.description, func(t *testing.T) {
+				realmClient := mock.RealmClient{}
+				realmClient.FindUsersFn = func(groupID, appID string, filter realm.UserFilter) ([]realm.User, error) {
+					return tc.users, nil
+				}
+
+				users, err := tc.inputs.resolveUsers(realmClient, "groupID", "appID")
+				assert.Nil(t, err)
+				assert.Equal(t, tc.expectedUsers, users)
+			})
+		}
+	})
+
+	for _, tc := range []struct {
+		description   string
+		users         []realm.User
+		expectedUsers []realm.User
+		expectedErr   error
+	}{
+		{
+			description: "should error when a user cannot be found from provided ids",
+			expectedErr: errors.New("no users found"),
+		},
+		{
+			description:   "should find users from provided ids",
+			users:         testUsers[:2],
+			expectedUsers: testUsers[:2],
+		},
+	} {
+		t.Run(tc.description, func(t *testing.T) {
+			realmClient := mock.RealmClient{}
+			realmClient.FindUsersFn = func(groupID, appID string, filter realm.UserFilter) ([]realm.User, error) {
+				return tc.users, nil
+			}
+
+			inputs := multiUserInputs{Users: []string{testUsers[0].ID, testUsers[1].ID}}
+			users, err := inputs.resolveUsers(realmClient, "groupID", "appID")
+
+			assert.Equal(t, tc.expectedUsers, users)
+			assert.Equal(t, tc.expectedErr, err)
+		})
+	}
+
+	t.Run("should error from client", func(t *testing.T) {
+		realmClient := mock.RealmClient{}
+		realmClient.FindUsersFn = func(groupID, appID string, filter realm.UserFilter) ([]realm.User, error) {
+			return nil, errors.New("client error")
+		}
+		inputs := multiUserInputs{}
+		_, err := inputs.resolveUsers(realmClient, "groupID", "appID")
+
+		assert.Equal(t, errors.New("client error"), err)
+	})
+}
+
+func TestSelectMultiUsersInputs(t *testing.T) {
+	testUsers := []realm.User{
+		{
+			ID:         "user-1",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeAnonymous}},
+		},
+		{
+			ID:         "user-2",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeUserPassword}},
+			Disabled:   true,
+			Data:       map[string]interface{}{"email": "user-2@test.com"},
+		},
+		{
+			ID:         "user-3",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeUserPassword}},
+			Disabled:   true,
+			Data:       map[string]interface{}{"email": "user-3@test.com"},
+		},
+	}
+
+	t.Run("should prompt for users", func(t *testing.T) {
+		for _, tc := range []struct {
+			description   string
+			inputs        multiUserInputs
+			procedure     func(c *expect.Console)
+			users         []realm.User
+			expectedUsers []realm.User
+		}{
+			{
+				description: "with no input set",
+				procedure: func(c *expect.Console) {
+					c.ExpectString("Which user(s) would you like to delete?")
+					c.Send("user-1")
+					c.SendLine(" ")
+					c.ExpectEOF()
+				},
+				users:         testUsers,
+				expectedUsers: []realm.User{testUsers[0]},
+			},
+			{
+				description: "with all input set, except users",
```

Comment:
does this test really do much different than the one before it?  I think what we need a test case here for is when `selectUsers` returns early and never prompts the user for input